docs/source/models/intervention_models/intrapartum/acs_intervention.rst

```diff
@@ -176,52 +147,8 @@ of ACS for pregnant women and people at imminent risk of preterm delivery.
     - see below
-    - see `Calibration strategy` section below for details on how to calculate PAF that is consistent with RR, risk exposure, and facility choice model
-
-Calibration Strategy
---------------------
-
-.. math::
-    \begin{align*}
-        p(\text{preterm with RDS}) 
-        &= \sum_{\text{paths without ACS}} p(\text{path})\cdot p(\text{preterm with RDS}|\text{no ACS})\\
-        &+ \sum_{\text{paths with ACS}} p(\text{path})\cdot p(\text{preterm with RDS}|\text{ACS})\\[.1in]
-        p(\text{preterm with RDS}|\text{no ACS}) &= \text{RR}_\text{no ACS} \cdot p(\text{preterm with RDS}|\text{ACS})
-    \end{align*}
-
-where :math:`p(\text{preterm with RDS})` is the probability of dying from preterm with RDS in the general population, and :math:`p(\text{preterm with RDS}|\text{ACS})` 
-and :math:`p(\text{preterm with RDS}|\text{no ACS})` are the probability of dying from preterm with RDS in setting with and without access to ACS.  
-For each path through the decision tree, :math:`p(\text{path})` is the probability of that path; for example the path that includes the edges 
-labeled BEmONC and unavailable occurs with probability that the birth is in a BEmONC facility times the probability that the facility has ACS 
-available.
-
-When we fill in the location-specific values for delivery facility rates, ACS coverage, relative risk of mortality with ACS access, 
-and mortality probability (which is also age-specific), this becomes a system of two linear equations with two unknowns (:math:`p(\text{preterm with RDS}|\text{ACS})` 
-and :math:`p(\text{preterm with RDS}|\text{no ACS})`), which we can solve analytically using the same approach as in the :ref:`cpap calibration <cpap_calibration>`. 
-
-**Alternative PAF Derivation**: An alternative, and possibly simpler derivation of the PAF that will calibrate this model comes from the observation that 
-:math:`\text{PAF} = 1 - \frac{1}{\mathbb{E}(\text{RR})}`.  If we define 
-
-.. math::
-
-   p(\text{no ACS}) = \sum_{\text{paths without ACS}} p(\text{path}),
-
-then can use this to expand the identity
-
-.. math::
-
-   \text{PAF}_\text{no ACS} = 1 - \frac{1}{\mathbb{E}(\text{RR})}.
-
-Since our risk exposure has two categories,
-
-.. math::
-
-   \mathbb{E}(\text{RR}) = p(\text{no ACS}) \cdot \text{RR}_\text{no ACS} + (1 - p(\text{no ACS})) \cdot 1.
-
-.. note:: 
-
-  Simulants should have the same propensity value for CPAP (see the :ref:`CPAP intervention model page <intervention_neonatal_cpap>`) and ACS, such that the same
-  simulants are exposed to both interventions (i.e., if coverage of both CPAP and ACS is 50%, then the same 50% of simulants will be getting each intevention).
+    - see the :ref:`CPAP intervention Calibration Strategy <intervention_neonatal_cpap>` for details on how to calculate PAF that is consistent with RR, risk exposure, and facility choice model.
+      No adjustments are needed to the PAF calculation for ACS, as it is the same as the CPAP intervention.
```

Comment:
I think along with Ali's suggestion above, this line should be removed; we are literally using one PAF for both, not using the same calculation method for each.